.gitignore

```diff
@@ -6,7 +6,7 @@
 *.so
 *.dylib
 *.nodes.json
-
+storage
```

Comment:
I think you should tell git that it must ignore only the directory inside the root of the project.

Reply:
`storage` is a default filename to store nodes' names